spec/models/phone_configuration_spec.rb

```diff
@@ -30,4 +30,31 @@
       end
     end
   end
+
+  describe '#masked_phone' do
+    let(:phone_configuration) { build(:phone_configuration, phone: phone) }
+    let(:phone) { '+1 703 555 1212' }
```

Comment:
In real-world, would this number be stored as dash-delimited?

```suggestion
    let(:phone) { '+1 703-555-1212' }
```

Reply:
It was copied from elsewhere in the file, and it gets parsed and reformatted anyways so I'm not super worried?